src/components/Home.jsx

```diff
@@ -31,12 +34,49 @@ function Home() {
     }
   };
 
+  const handleTokenSubmit = async () => {
+    //create query for token collection in firestore
+    const q = query(collection(db, 'token'), where('token', '==', userToken));
+
+    const querySnapshot = await getDocs(q);
+
+    //set token in local storage if it exists in firebase firestore
```

Comment:
Another great comment, explains what is next block of code meant to do so "reader" doesn't need to parse each line of code to get a sense what it's meant to do.